eodag/rest/server.py

```diff
@@ -271,42 +273,54 @@ def stac_extension_oseo(request: Request):
     return jsonable_encoder(response)
 
 
+class SearchBody(BaseModel):
+    """
+    class which describes the body of a search request
+    """
+
+    provider: Union[str, None] = None
+    collections: List[str]
+    datetime: Union[str, None] = None
+    bbox: Union[str, None] = None
```

Comment:
```suggestion
    bbox: Union[list, str, None] = None
```
Handle bbox passed as a list (STAC API default) or as a simple str, see https://github.com/radiantearth/stac-api-spec/tree/main/item-search#query-parameter-table